test/trip_dispatch_test.rb

```diff
@@ -1,72 +1,74 @@
 require_relative 'test_helper'
```

Comment:
you dropped the "er" in the title of this file. It should be `trip_dispatcher_test.rb.` Sticking with this convention is important for the use of guard, and it will be very important in Rails to pay attention to naming coventions.